src/main/scala/com/typesafe/sbt/license/TargetLanguage.scala

```diff
@@ -77,3 +77,19 @@ case object Html extends TargetLanguage {
 
   def htmlEncode(s: String) = org.apache.commons.lang3.StringEscapeUtils.escapeHtml4(s)
 }
+
+case object Csv extends TargetLanguage {
+  val ext = "csv"
+  def documentStart(title: String, reportStyleRules: Option[String]): String = ""
+  def documentEnd(): String = ""
+  def createHyperLink(link: String, content: String): String = s"$content"
```

Comment:
Hmm. That concern is certainly valid. I suppose that a pair of type `(Description, URL)` actually maps better onto two columns than onto one. However, trying to get the plugin code to produce two columns for (just) this format would probably necessitate a far more invasive restructuring. I guess the question is, what's the worse evil? Leaving off the license url entirely or smooshing two (logical) columns into one... I guess the answer to that question depends on what we are optimizing for. If it's tools, than probably smooshing the columns is worse, as we may want to, e.g., match licenses on their names, not on their `(Name, URL)` pairs. However, if we are optimizing for consumption by humans, who likely want to see the URLs, dropping them is probably worse.